scripts/kind-dev-env.sh

```diff
@@ -23,16 +23,10 @@ SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 : "${IMAGE_REGISTRY:=ghcr.io/llm-d}"
 
 # Set a default VLLM_SIMULATOR_IMAGE if not provided
-: "${VLLM_SIMULATOR_IMAGE:=llm-d-inference-sim}"
-
-# Set a default VLLM_SIMULATOR_TAG if not provided
-export VLLM_SIMULATOR_TAG="${VLLM_SIMULATOR_TAG:-latest}"
+export VLLM_SIMULATOR_IMAGE="${VLLM_SIMULATOR_IMAGE:-ghcr.io/llm-d/llm-d-inference-sim:latest}"
```

Comment:
Please set the default tag using the VLLM_SIMULATOR_TAG environment variable and restore the prior line